src_bagl/ui_flow_signMessage712.c

```diff
@@ -1,62 +1,87 @@
-#include "shared_context.h"
-#include "ui_callbacks.h"
+#ifdef HAVE_EIP712_FULL_SUPPORT
 
-void prepare_domain_hash_v0() {
-    snprintf(strings.tmp.tmp, 70, "0x%.*H", 32, tmpCtx.messageSigningContext712.domainHash);
-}
+#include "ui_logic.h"
+#include "shared_context.h"  // strings
+
+enum { UI_712_POS_REVIEW, UI_712_POS_END };
```

Comment:
Not a fan of single line enums
Can we change this in clang format if it's not too much changes ?

Reply:
Will look into choosing a better clang-format configuration in a future PR. :+1: